Makefile

```diff
@@ -2,7 +2,7 @@ DESTDIR   ?=
 PREFIX    ?= /usr/local
 MANPREFIX ?= $(PREFIX)/man
 CFLAGS += -Wall -Wextra
-LDLIBS += -lcurses -ltinfo
+LDLIBS += `pkg-config --libs ncurses 2>/dev/null || echo '-lcurses -ltinfo'`
```

Comment:
this will not work on a system that doesn't have pkg-config

Reply:
@tenox7 how will it not? That's what the `||` is for.

PS: Having a build require pkg-config would not be something crazy to ask from users also. It's about as standard of a build tool as it gets.